src/accelerate/commands/config/cluster.py

```diff
@@ -50,6 +50,39 @@ def get_cluster_input():
                 "What is the port you will use to communicate with the main process? ",
                 lambda x: int(x),
             )
+
+    deepspeed_config = None
+    if distributed_type in [DistributedType.MULTI_GPU, DistributedType.NO]:
+        use_deepspeed = _ask_field(
+            "Do you want to use DeepSpeed (yes/no)? [NO]",
```

Comment:
```suggestion
            "Do you want to use DeepSpeed (yes/no)? [no]",
```

Reply:
I think you probably want a space at the end of this to be consistent with the other _ask_fields

Reply:
Should actually be 
```
          "Do you want to use DeepSpeed? [yes/NO]: ",
```
to be compatible with what is actually in the lib for now.